wiki/People/The_Team/Global_Moderation_Team/th.md

```diff
@@ -3,64 +3,63 @@ tags:
 
-โดยทั่วไปทีมงานจะสังเกตผู้คนเป็นเวลาหลายสัปดาห์เพื่อให้แน่ใจว่าพวกเขาเป็นสมาชิกที่มีค่าของทีม โปรดอดทน และ อย่าถามใครจาก **GMT** เกี่ยวกับกฎการรายงานผู้ใช้และปัญหาอื่น ๆ
+โดยทั่วไป ทีมงานจะสังเกตผู้คนเป็นเวลาหลายสัปดาห์เพื่อให้แน่ใจว่าพวกเขาเป็นสมาชิกที่มีค่าต่อทีม โปรดอดทน และอย่าทำตัวเกินเลย ทั้งนี้ สามารถถามใครก็ได้จาก GMT เกี่ยวกับกฎ การรายงานผู้ใช้ และปัญหาอื่น ๆ
 
 ### บางสิ่งที่ไม่ควรทำ
 
-1. อย่าถาม **GMT** เกี่ยวกับสถานะของคุณในฐานะผู้สมัคร **GMT** จะไม่เพิ่มทุกคนหรอก ถ้าหากมีใครถามนั้นก็หมายความว่าการสมัครของคนนั้นจะสิ้นสุดทันที ความคิดของผู้อื่นนั้นเป็นกุญแจสำคัญ
-2. อย่ารายงานทุกสิ่งที่เป็นสิ่งเล็กน้อยในการแชท มันเป็นทักษะที่สำคัญในการรู้ว่าอะไรที่ควรรายงาน หรือ ไม่ควร
-3. อย่าตอบคำถามของผู้อื่นโดยไม่รู้ว่าคำตอบนั้นถูกต้อง หรือ ไม่โดยเฉพาะในฟอรัม ข้อเสนอแนะนั้นก็ดี แต่คนที่เข้าใจผิดมันจะไม่เป็นประโยชน์เลย หากคุณไม่ทราบวิธีแก้ปัญหา ควรให้ผู้อื่นตอบ และ เรียนรู้จากพวกเขา
-4. อย่าหักโหมสิ่งต่าง ๆ ช่วยเหลือเพียงพอดีไม่ใช่จำนวนที่มากเกินไป เรื่องนี้เป็นที่รู้กันดีว่าจะทำให้หมดความอดทน และ ความมุ่งมั่น ทำมันให้ง่ายๆ สิ!
+1. อย่าถาม GMT เกี่ยวกับสถานะของคุณในฐานะผู้สมัคร โดย GMT จะไม่เพิ่มคนที่มีส่วนร่วมเพื่อประโยชน์ของตนเองเท่านั้น การที่ไม่เห็นแก่ตัวเป็นกุญแจสำคัญ
+2. อย่ารายงานทุกสิ่งเล็กน้อยในการแชท เป็นทักษะที่สำคัญที่จะรู้ว่าอะไรที่ควรหรือไม่ควรรายงาน
+3. อย่าตอบคำถามของผู้อื่นโดยไม่รู้ว่าคำตอบนั้นถูกต้องหรือไม่ โดยเฉพาะในฟอรัม ข้อเสนอแนะนั้นก็ดี แต่การทำให้คนเข้าใจผิดนั้นจะไม่เป็นประโยชน์เลย หากคุณไม่ทราบวิธีแก้ปัญหา ควรให้ผู้อื่นตอบ และเรียนรู้จากพวกเขา
+4. อย่าหักโหมสิ่งต่าง ๆ ช่วยเหลือเพียงพอดี ไม่ใช่จำนวนที่มากเกินไป เรื่องนี้เป็นที่รู้กันดีว่าจะทำให้หมดความอดทนและความมุ่งมั่น ทำตัวสบาย ๆ!
 
 ## สมาชิกทีม
 
 [กลุ่มทีมผู้ดูแล](https://osu.ppy.sh/groups/4) รายชื่อสมาชิกทั้งหมดของทีมผู้ดูแล
 
-*หมายเหตุ: สมาชิก **GMT**  ทั้งหมดพูดภาษาอังกฤษยกเว้นที่ระบุไว้เป็นอย่างอื่น*
+*หมายเหตุ: สมาชิก **GMT** ทั้งหมดพูดภาษาอังกฤษยกเว้นที่ระบุไว้เป็นอย่างอื่น*
 
 | รายชื่อ | ภาษาที่ใช้ | ตำแหน่ง/หน้าที่ |
 | :-- | :-- | :-- |
 | ![][flag_MY] [- [bibitaru] -](https://osu.ppy.sh/users/4482419) | ภาษาจีน และ ภาษามาเลเซีย | ผู้ดูแลการแชท |
 | ![][flag_DE] [- Felix](https://osu.ppy.sh/users/8503985) | ภาษาเยอรมัน | ผู้ดูแลการแชท |
-| ![][flag_US] [abraker](https://osu.ppy.sh/users/4635891) | ภาษาอังกฤษ | ผู้ดูแลฟอรัม |
-| ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) | ภาษาอังกฤษ | ผู้ดูแลการแชท, ผู้ดูแลบีทแมพ |
+| ![][flag_US] [abraker](https://osu.ppy.sh/users/4635891) |  | ผู้ดูแลฟอรัม |
+| ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) |  | ผู้ดูแลการแชท, ผู้ดูแลบีทแมพ |
 | ![][flag_DE] [Clobohne](https://osu.ppy.sh/users/499343) | ภาษาเยอรมัน | ผู้ดูแลการแชท |
 | ![][flag_AR] [Darksonic](https://osu.ppy.sh/users/570042) | ภาษาสเปน | ผู้ดูแลการแชท, ผู้ดูแลฟอรัม |
-| ![][flag_US] [Death](https://osu.ppy.sh/users/3242450) | ภาษาอังกฤษ | ผู้ดูแลการแชท, ผู้สนับสนุนเทคนิคต่างๆ, ผู้ดูแลบีทแมพ |
+| ![][flag_US] [Death](https://osu.ppy.sh/users/3242450) |  | ผู้ดูแลการแชท, ผู้สนับสนุนเทคนิคต่างๆ, ผู้ดูแลบีทแมพ |
```

Comment:
```suggestion
| ![][flag_US] [Death](https://osu.ppy.sh/users/3242450) |  | ผู้ดูแลการแชท, ผู้สนับสนุนทางเทคนิคต่าง ๆ , ผู้ดูแลบีทแมพ |
```

Apply to all the ones with ผู้สนับสนุนทางเทคนิคต่าง ๆ
Im not sure that it should be "ต่าง ๆ," or "ต่าง ๆ ,"

Reply:
Actually removing ต่าง ๆ might be a good idea

Reply:
``` yaml
    support: ผู้สนับสนุนทางเทคนิค
```
Already changed as part of #4462, I didn't touch the table here other than these changes

Reply:
Huh neat. I will go ahead and review #4462 when Im free then